addons/account/models/account_move_line.py

```diff
@@ -1631,10 +1624,6 @@ def write(self, vals):
             if line.parent_state == 'posted' and any(self.env['account.move']._field_will_change(line, vals, field_name) for field_name in protected_fields['tax']):
                 line._check_tax_lock_date()
 
-            # Check the reconciliation.
-            if any(self.env['account.move']._field_will_change(line, vals, field_name) for field_name in protected_fields['reconciliation']):
-                line._check_reconciliation()
-
```

Comment:
What is protecting against this?
* Create invoice of 1000€
* Register full payment
* -> there are now partial records for a value of 1000
* -> there is also a full record, and the matching number is for fully paid
* Now modify the invoice to 9999999999€ (or change the currency, or any protected field)

What happens to the reconciliation?